udaykalra/portfolio/src/main/webapp/style.css

```diff
@@ -124,7 +139,6 @@ div.main-content {
   background-color: var(--body-color);
   color: var(--main-color);
   font: 'Roboto', sans-serif;
-
   flex: 1;
```

Comment:
Sort alphabetically?

Also the html, body rules below.

Reply:
Done